### PR TITLE
content/quickstart: use open source exporters for Go quickstarts

### PR DESCRIPTION
**Pull request by odeke-em**
Use free and open source exporters for Go quickstarts:
* Prometheus for Metrics
* Zipkin for Tracing

This change will be replicated across most/all languages that
support the above exporters. It makes the project more attractive,
easier to setup and non-biased towards any company.

Updates #343

### CODE REVIEW

File: content/quickstart/go/metrics.md [hunk 1]
```diff
@@ -1335,227 +1415,67 @@ func processLine(ctx context.Context, in []byte) (out []byte, err error) {
...
-With the above you should now be able to navigate to the [Google Cloud Platform console](https://app.google.stackdriver.com/metrics-explorer), select your project, and view the metrics.
+and then return to the terminal that's running the Go metrics tutorial and generate some work by typing inside it and it will look something like:
+![](/images/metrics-prometheus-sample-repl.png)
 
-In the query box to find metrics, type `quickstart` as a prefix:
+## Viewing your metrics
+With the above you should now be able to navigate to the Prometheus UI at http://localhost:9090
 
-![viewing metrics 1](https://cdn-images-1.medium.com/max/1600/1*kflo3l46PslT6oZDNCJ23A.png)
+which  should firstly show:
```
Comment (songy23) [inline content/quickstart/go/metrics.md]: nit: there's one redundant space here.

File: content/quickstart/go/tracing.md [hunk 2]
```diff
@@ -339,252 +278,52 @@ func processLine(ctx context.Context, in []byte) (out []byte, err error) {
...
-To get our code ready to export, we will be adding a few lines of code to our `main` function.
-
-1. We want to make our traces export to Stackdriver
-```go
-stackdriver.NewExporter
-trace.RegisterExporter
 ```
 
-2. We want to trace a large percentage of executions (this is called [sampling](/core-concepts/tracing/#sampling))
+2. For demo purposes we would like to see the largest percentage of traces -- this is called [sampling](/core-concepts/tracing/#sampling)
```
Comment (songy23) [inline content/quickstart/go/tracing.md]: Maybe also add a note that in production users should not always sample?

File: content/quickstart/go/tracing.md [hunk 3]
```diff
@@ -710,14 +452,37 @@ func processLine(ctx context.Context, in []byte) (out []byte, err error) {
...
+Having already succesfully started Zipkin as in [Zipkin Codelab](/codelabs/zipkin), we can now run our code by
+
+```shell
+go run repl.go
+```
+
+## Viewing your traces
+With the above you should now be able to navigate to the Zipkin UI at http://localhost:9411
+
+which will produce such a screenshots
```
Comment (songy23) [inline content/quickstart/go/tracing.md]: nit: s/screenshots/screenshot

**Comment by odeke-em** [comment]
@songy23 I've updated with your suggestions with the exception of advising how to sample in production, because I feel like we'd have to explain why we don't always sample: the high storage and overhead costs which then detracts from the quickstart unfortunately. That advice can be handled in the FAQs and on the tracing concepts page.

**Comment by odeke-em** [comment]
Thank you for the review @songy23!

**Pull request merged**